NWEEI/NWEEI/Views/About/History.cshtml

```diff
@@ -0,0 +1,77 @@
+﻿@{
+    ViewData["Title"] = "Mission & History";
+}
+<div class="container content-section">
+    <h3>Mission & History</h3>
+    <p>
+        The Northwest Water & Energy Education Institute (NWEEI) offers both standard and custom learning opportunities throughout the Northwest for energy and water industry practitioners – including an energy management certification program. NWEEI is located within the Science Department at Lane Community College.  Starting in 2018 all Energy Classes have been offered online and in Fall of 2019 the Water Conservation Program is also online.  Great opportunities for folks outside of the Lane Community College area.
+    </p>
+</div>
+
+<div class="dark-block">
+    <div class="container">
+        <div class="content-section">
+            <p>
+                Guided by a regional advisory board, NWEEI has provided energy efficiency training and development throughout the Northwest region since 1998. Participants have included technicians, contractors, engineers, architects, consultants, utility personnel, and energy managers.
+            </p>
+            <p>
+                The Energy Management Program offered through Lane Community College has ties back as far as 1980 when Alan Gubrud, a physics instructor, obtained a grant from the National Science Foundation to develop a curriculum that combined the principles of basic physics with energy analysis techniques. Students gained a working understanding of energy systems in the built environment and the tools to analyze and quantify energy efficiency efforts. The program began with an emphasis on residential energy efficiency / general solar energy systems and has evolved to include technician level commercial energy efficiency, renewable energy system installation and water conservation skillsets.
+            </p>
+            <br />
```

Comment:
If you're not attached to this br, I say we kill it in the name of consistent spacing :)